pypesto/visualize/misc.py

```diff
@@ -359,6 +359,13 @@ def process_start_indices(
         if start_index < len(result.optimize_result)
     ]
 
+    # filter out the indices that are not finite
+    start_indices = [
+        start_index
+        for start_index in start_indices
+        if np.isfinite(result.optimize_result[start_index].fval)
+    ]
```

Comment:
Should we check for `np.isnan` here as well?

Reply:
np.isfinite tests for `Not a Number` already.